package.json

```diff
@@ -150,7 +152,7 @@
     "semver": "^6.3.0",
     "send": "^0.17.1",
     "shelljs": "^0.8.3",
-    "stylelint": "^13.2.0",
+    "stylelint": "^13.3.1",
```

Comment:
Should we add an explicit dependency on `postcss` here? Even though it's transitively brought in, since we explicitly access it, I think it might make sense to add it. Yarn could hoist the package under `stylelint` and the rules will fail.

Reply:
I don't think we actually import `postcss` directly anywhere, we just deal with the AST that Stylelint gives us.

Reply:
I see it in the `no-ampersand-beyond-selector-start.ts` file.

Reply:
I meant for concrete imports. The one there is used for the type info. Either way, I don't mind adding it.

Reply:
It will still fail within the `ts-node` compilation though. I don't mind too much. I just remember that we ran into issues with something similar in the past. Up to you.